torch/sparse/__init__.py

```diff
@@ -45,15 +45,15 @@ def addmm(mat: Tensor, mat1: Tensor, mat2: Tensor,
 def mm(mat1: Tensor, mat2: Tensor) -> Tensor:
     r"""
     Performs a matrix multiplication of the sparse matrix :attr:`mat1`
-    and dense matrix :attr:`mat2`. Similar to :func:`torch.mm`, If :attr:`mat1` is a
+    and matrix :attr:`mat2`. Similar to :func:`torch.mm`, If :attr:`mat1` is a
```

Comment:
"and matrix" -> "and the (sparse or strided) matrix"

Reply:
What's the format of the output tensor of this function when it's sparse x sparse and when it's sparse x dense?

Reply:
The format of the output tensor follows:  
`sparse x sparse -> sparse`
`sparse x dense -> dense` 

Let me add this to the doc. in order to be clear to the user.